build.sbt

```diff
@@ -87,6 +87,17 @@ lazy val `scalafmt-sbt` = project
   )
   .dependsOn(cli)
 
+lazy val `scalafmt-sbt` = project
+  .settings(
+    allSettings,
+    mimaPreviousArtifacts := Set.empty,
+    moduleName := "sbt-scalafmt",
+    isOnly(scala212),
+    sbtPlugin := true,
+    sbtVersion in Global := "1.0.0"
```

Comment:
Don't put settings `in Global` in the settings of a `project`. It is extremely confusing as that affects the whole build, not just the project. It makes no sense.

Reply:
old plugin has this setting too :)

Reply:
Then fix the old plugin too ;)

Reply:
This is effectively defining which sbt version the plugin targets, rather than defaulting whatever sbt version you're using for the whole build as defined in `build.properties`. I've used this in the past to ensure my plugin works on old versions of sbt while using the latest version to build the plugin.